os/arch/arm/src/imxrt/imxrt_mpuinit.c

```diff
@@ -85,10 +85,10 @@
 const struct mpu_region_info regions_info[] = {
 #if defined(CONFIG_SYSTEM_PREAPP_INIT) && !defined(CONFIG_APP_BINARY_SEPARATION)
 	{
-		&mpu_userflash, (uintptr_t)__uflash_segment_start__, (uintptr_t)__uflash_segment_size__, MPU_REG_USER_CODE,
+		&mpu_userflash, (uintptr_t)__uflash_segment_start__, (uintptr_t)__uflash_segment_size__, 0,
```

Comment:
Why do we need to keep the region number manually?
Originally mpu function assigns the number automatically. But to know the number of some specific region, we modified to pass the region number. but currently it looks unnecessary and assignment automatically is better to use.

Reply:
As you know, we still need to know the region number assigned to every region.